### ISSUE TITLE
Profile Dashboard: Add application overview doc

### ISSUE THREAD

**Issue by humancompanion** [USAJOBS/Help#19]
We need a document to act as the target of the Help link in each application drawer. This doc should cover the entire contents of the drawer (face and inside contents). It should also include, or link to a doc on, the archiving process and business rules around that process.

**Comment by dalenard21** [comment]
@humancompanion @PeterToohey 

We already have content that covers:

**How to understand the job announcement** - this covers who may apply, location, closing date, duties, requirements, qualifications, etc.  Most of the things that are displayed in the "face" or top part of the application drawer are covered by this content.

**How to continue an application** - this content covers how to continue/complete an application that's in progress. However, we need to update the content because the CTA in the new profile dashboard is "Complete Application", not 'Continue Application". 

**How to check the status of an application** - this content covers the various application statuses which are located within the face of the drawer

The only content we DON'T have is what it means to, and how to, archive applications.

**Question**
Do we need a new how to page to explain what's in the application drawer?  

A new page would be repetitive of some how to content we already have (listed above).  Can the help icon link to the How To landing page (as long as we add the how to archive content)? Or is there a way to pull in the related How To's we already have and create a page with Application and Job Announcement how-tos only?

Happy to discuss further if this isn't clear.

**Comment by humancompanion** [comment]
@dalenard21 - I'm not comfortable with the help icon not having a distinct target. The page could be a lot like the how-to/search/ page in that it basically just links elsewhere. Basically, the overview you just provided in the previous comment would be fine plus the archiving document.

**Comment by dalenard21** [comment]
@humancompanion , @PeterToohey 

I emailed you a word doc with the content for how to understand the application listing/drawer.  Would like a quick review to make sure I covered everything that's needed. 

Where should this content live?  Is it okay to create the page in /help-center/how-to/application ?

Let me know and I can create it.

**Comment by humancompanion** [comment]
@dalenard21 - Let's hold of on creating it until after 5.5 is wrapped up (we could branch but we don't have a deploy target) but, I do need to know the URL it will eventually live at so I can put that into the prototype. Are we using /Help/how-to/application/ and moving the current content to a different URL?

**Comment by humancompanion** [comment]
@dalenard21 - I created this page at how-to/application/view/. I did add it to the how-to index because I want that to be as complete as possible **for sections that we have updated**. Also, because it is a jump page it is helpful to have at that top level. It does a good job of providing an overview.

The only alteration I made to the content itself was to change "complete an application" to "continue an application" to be consistent with the title of the page we're linking to and with the UI of profile dashboard (we use continue when the app is in the "In progress" state).
